proof/refine/RISCV64/KHeap_R.thy

```diff
@@ -6,80 +6,174 @@
 
 theory KHeap_R
 imports
+  "AInvs.AInvs"
```

Comment:
Remove `AInvs` re-import here, it is already imported and will destroy any simpset deletion setup from before.